src/settings-ui/PowerToys.Settings/App.xaml.cs

```diff
@@ -13,13 +14,25 @@ namespace PowerToys.Settings
     /// </summary>
     public partial class App : Application
     {
+        private MainWindow mainWindow;
+
         public bool ShowOobe { get; set; }
 
+        public void OpenMainWindow(Type type)
+        {
+            if (mainWindow == null)
+            {
+                mainWindow = new MainWindow();
+            }
+
+            mainWindow.OpenMainWindow(type);
```

Comment:
If you call `mainWindow.Show();` before calling `mainWindow.OpenMainWindow(type);` it will set the `inst` variable and not require the following code in `MainWindow.OpenMainWindow(Type type)`
```
else
            {
                var newWindow = new MainWindow();
                newWindow.Show();
            }
```